docs/build/fluence-js/changelog.md

```diff
@@ -2,27 +2,193 @@
 
 Fluence JS versioning scheme is the following: `0.BREAKING.ENHANCING`
 
-* `0` shows that Fluence JS does not meet its vision yet, so API can change quickly
-* `BREAKING` part is incremented for each breaking API change
-* `ENHANCING` part is incremented for every fix and update which is compatible on API level
+- `0` shows that Fluence JS does not meet its vision yet, so API can change quickly
+- `BREAKING` part is incremented for each breaking API change
+- `ENHANCING` part is incremented for every fix and update which is compatible on API level
 
-## [0.20.2](https://github.com/fluencelabs/fluence-js/releases/tag/v0.19.1) – February 23, 2022
+## [0.26.3](https://github.com/fluencelabs/fluence-js/releases/tag/0.26.3) - Oct 26, 2022
+
+**Fluence Peer:**
+
+Implement new builtin to work with JSON-like structures ([#195](https://github.com/fluencelabs/fluence-js/pull/195))
+
+## [0.26.2](https://github.com/fluencelabs/fluence-js/releases/tag/0.26.2) - Oct 9, 2022
+
+Update AVM to 0.30.11 ([#191](https://github.com/fluencelabs/fluence-js/pull/191))
+
+## [0.26.1](https://github.com/fluencelabs/fluence-js/releases/tag/0.26.1) - Oct 9, 2022
+
+**Compiler support:**
+
+Add v4 version of compiler support. It makes possible to use any name for functions and data structures in aqua without colliding with variable names in FluenceJS ([#190](https://github.com/fluencelabs/fluence-js/pull/190))
```

Comment:
Add v4 of compiler support. It makes it possible to use any name for functions and data structures in aqua without collisions with variable names in FluenceJS